lib/util/stream_utils.js

```diff
@@ -108,10 +108,8 @@ shaka.util.StreamUtils = class {
         const currentCodec =
           MimeUtils.getNormalizedCodec(stream.codecs);
         if (previousCodec == currentCodec) {
-          if (stream.bandwidth > previousStream.bandwidth) {
-            validAudioStreams.push(stream);
-            validAudioIds.push(stream.id);
-          }
+          validAudioStreams.push(stream);
```

Comment:
This means we would no longer be filtering duplicate bandwidths at all, even when that information is available.

Instead it might be better to just have a specific exception for unknown bandwidth.

```js
if (!stream.bandwidth || stream.bandwidth > previousStream.bandwidth) {
```

Reply:
Done!